### ISSUE TITLE
iocage fetch broken for un-official plugins

### ISSUE THREAD

**Issue by jsegaert** [iocage/iocage#526]
Make sure to follow and check these boxes before submitting an issue! Thank you.

- [X] Supply `iocage --version`
```
/home/jurgen/iocage jurgen@trinity%  git log -1
commit 1908813cbd01c6770ced228f4c20c8ba80f3dad9 (HEAD -> master, upstream/master, origin/master, origin/HEAD)
Author: Brandon Schneider <brandon@ixsystems.com>
Date:   Wed Apr 11 17:23:05 2018 -0500
```

- [X] Supply the commands used, along with any steps to recreate it.
```
/home/jurgen jurgen@trinity%  sudo iocage fetch -P -n mineos.json ip4_addr="wlan0|192.168.0.210"
```
- [X] Provide the output from the command you issued.
```
/home/jurgen jurgen@trinity%  sudo iocage fetch -P -n mineos.json ip4_addr="wlan0|192.168.0.210"
Plugin: mineos
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/fetch.py", line 136, in cli
    ioc.IOCage(exit_on_error=True).fetch(**kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/iocage.py", line 973, in fetch
    ).fetch_plugin(name, props, 0, accept)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_plugin.py", line 115, in fetch_plugin
    self.__fetch_plugin_inform__(conf, num, plugins, accept_license)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_plugin.py", line 144, in __fetch_plugin_inform__
    "message": f"  Official Plugin: {conf['official']}"
KeyError: 'official'
```
- [X] Supply what you expected the result or output to be
Unofficial plugin created from JSON manifest file.  :-)
This used to work, but was broken by one of the recent commits that added the "official" property to the plugins INDEX file.
I'll take a look and may send a PR with the fix.
- [X] Checked that the problem has not already been fixed on `master` if using
 a stable release.

**Comment by skarekrow** [comment]
It seems I missed a spot with grabbing that key with `get`. Well spotted ;)

**Comment by jsegaert** [comment]
Fixed with #527
